tidy3d/components/source.py

```diff
@@ -256,14 +256,28 @@ class GaussianPulse(Pulse):
     >>> pulse = GaussianPulse(freq0=200e12, fwidth=20e12)
     """
 
+    allow_dc_component: bool = pydantic.Field(
```

Comment:
Minor point, but I think I'd slightly prefer `remove_dc_component: bool = pydantic.Field(True, ...)` because it's more in line with what we do (add a correction to remove the DC component).